### ISSUE TITLE
When I create a movie from phone with resolution 1080p, the video is very small and even in full screen. But "original" is fine.

### ISSUE THREAD

**Issue by Anthrwpos** [Helioviewer-Project/helioviewer.org#427]
Need to make any change?

**Comment by dgarciabriseno** [comment]
I think you're problem is the same that was reported here in #285 

I think it's worth taking a deeper look, but I might end up with the same response once I understand why it's coming out that way.

**Comment by Anthrwpos** [comment]
Τhree years ago it says low resolution but from the page on PC version not from mobile, from PC it's fine.

How does this relate to the issue I'm bringing up from mobile and HD resolution on a very small display? Thanks

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by dgarciabriseno** [comment]
You're right, it seems different because the mobile issue isn't being stretched, the opposite on mobile.
It's related because the code that creates the movies is the same between PC and Mobile, the only difference is the screen size ("viewport size" in helioviewer terms). I need to take a closer look to see what's happening.

**Comment by dgarciabriseno** [comment]
Ok, I've figured out what's going on by putting testing it on a desktop screen.

Here's an original sized mobile movie:
![image](https://user-images.githubusercontent.com/94071409/211942522-64da23e4-bf48-4981-9830-037f7e159c48.png)

And here's a "1080p" movie
![image](https://user-images.githubusercontent.com/94071409/211942443-8602d62f-8944-48be-aa81-815a91665765.png)

What's happening is that 1080p assumes a 1920x1080 resolution. The mobile video has a width and height of 400x900. small width, big height. A phone screen.
When you select 1080p, the resolution of the video becomes 1920x1080. Big width, small height. Most of that width is wasted and just becomes black because the viewport, the part that's rendered in the video, still has the mobile aspect ratio (400x900, but the 900 is stretched up to 1080).

So now when you go to play the movie on your phone, the invisible black parts are still part of the movie. Your phone wants to display the full 1920x1080 movie on your screen which is why it's super small looking. But only the middle of that 1920 width has content.
Helioviewer tries to do the same thing, so it also looks small there too.


I think what needs to happen is that when the other resolutions are chosen, the created video should maintain the viewport's aspect ratio.

**Comment by Anthrwpos** [comment]
Thanks for the reply, so, from phone is there a solution for this or it's like that?

 And I noticed another thing (or at least it used to work and now it doesn't) when a movie is created and the the white box appeared as the result of the video, when I click (from PC) or touch (from phone) the white box, the video doesn't open and need to go right up where all videos is. I think this used to open if I'm not mistaken and it showed the video directly without going to the top right where the videos are created. Thanks


________________________________

**Comment by Anthrwpos** [comment]
So, is there any solution about the resolution 1080 to project it as the  "original' selection?

**Comment by dgarciabriseno** [comment]
There's no solution yet. I haven't looked into it much further since my last reply.
